lib/edamam_api_wrapper.rb

```diff
@@ -0,0 +1,85 @@
+require 'httparty'
+
+class EdamamApiWrapper
+   #example url search: "https://api.edamam.com/search?q=chicken&app_id=${YOUR_APP_ID}&app_key=${YOUR_APP_KEY}&from=0&to=3&calories=591-722&health=alcohol-free"
```

Comment:
Nice work not giving away your app key, and keeping handy data nearby. Others goofed on this.